### PR TITLE
Fix that Time of Day2's keynote is hidden a little

### LINKED ISSUE #1094: Time of Day2's keynote is hidden a little
**Idea Description**

**Reference images and links**
image

### PR DESCRIPTION
**Pull request by ochim**
## Issue
- close #1094

## Overview (Required)
- Fix that Time of Day2's keynote is hidden a little
change the TimetableGridItem

## Screenshot (Optional if screenshot test is present or unrelated to UI)
Before | After
:--: | :--:
 |

### CODE REVIEW

File: feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/TimetableGridItem.kt [hunk 1]
```diff
@@ -123,7 +123,7 @@ fun TimetableGridItem(
 
             Row(
                 modifier = Modifier
-                    .weight(1.5f, fill = false)
+                    .weight(2f, fill = false)
```
Comment (takahirom) [inline feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/TimetableGridItem.kt:126]: Thanks! Do you have any idea why this works 👀

Comment (ochim) [reply feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/TimetableGridItem.kt:126]: Previously the Row height was not enough, so I tried increasing this value

Comment (takahirom) [reply feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/TimetableGridItem.kt:126]: Sorry for bothering you. But Could you add some preview functions to confirm small items? Maybe we can pass modifier.size or something to confirm.

Comment (ochim) [reply feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/TimetableGridItem.kt:126]: I have no idea. I think the existing preview functions are enough.

Comment (ochim) [reply feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/TimetableGridItem.kt:126]: By the way, the following code works also. 
```
            Text(
                modifier = Modifier.weight(1f, fill = false), // notice 
                text = timetableItem.title.currentLangTitle,
                style = titleTextStyle,
                overflow = TextOverflow.Ellipsis,
            )

            Row(
                modifier = Modifier
                    .weight(1f, fill = false) // notice
                    .padding(top = TimetableGridItemSizes.titleToSchedulePadding),
            ) {
               ...
            }
```

Comment (ochim) [reply feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/TimetableGridItem.kt]: Hello, I did [a30a61e](https://github.com/DroidKaigi/conference-app-2023/pull/1104/commits/a30a61e2ceec4a06a9aa1e13c556c98c6a8ad3a1) @takahirom

**Comment by takahirom** [comment]
Looks great! Thanks!

**Review by takahirom — APPROVED** [review]

**Pull request merged**